Server/server_message_constants.py

```diff
@@ -7,6 +7,7 @@ class MessageType(str, Enum):
     YOUR_ID = "your ID is..."
     FIRST_PLAYER = "you are the host"
     GAME_STARTED = "gl hf"
+    PLAYER_MOVEMENT = "movement of players"
```

Comment:
It is not movement. We send positions! We could send only movements, but then change the code to do that.